src/renderer/gl_engine/tvgGlRenderer.h

```diff
@@ -133,7 +133,7 @@ class GlRenderer : public RenderMethod
         Key key;
     } mDisposed;
 
-    bool mClearBuffer = true;
+    bool mClearBuffer = true;  //FIXME: clear buffer should be optional (default is false)
```

Comment:
@RuiwenTang It is not always necessary to clear the buffer if the scene has solid background content. The user can choose whether to do so.